doc/src/sgml/bgworker.sgml

```diff
@@ -427,8 +427,10 @@ postmasterがバックグラウンドワーカを開始しようと試みたか
    notifications with the <command>LISTEN</command> command, as there is no
```

Comment:
```
$git diff REL_13_1 REL_14_0 bgworker.sgml 
diff --git a/doc/src/sgml/bgworker.sgml b/doc/src/sgml/bgworker.sgml
index 7fd673a..8995191 100644
--- a/doc/src/sgml/bgworker.sgml
+++ b/doc/src/sgml/bgworker.sgml
@@ -134,8 +134,8 @@ typedef struct BackgroundWorker
 
   <para>
    <structfield>bgw_restart_time</structfield> is the interval, in seconds, that
-   <command>postgres</command> should wait before restarting the process, in
-   case it crashes.  It can be any positive value,
+   <command>postgres</command> should wait before restarting the process in
+   the event that it crashes.  It can be any positive value,
```
 in case it crashesがin the event that it crashesになっていますので訳出した方が良さそうです。

プロセスがクラッシュした場合に
→
プロセスがクラッシュするイベントでは

などでしょうか。

Reply:
in the event thatは、熟語で、意味としては"in case"とほとんど同じですが、in the event thatは「普通はないけど万が一の場合には」のようなニュアンスがあるようです。プロセスのクラッシュという、めったに起きない事象だから、ということでin the event that にしたのではないでしょうか。日本語に訳す場合はin caseと同じで良いと思いますが、あえて強調するなら「万が一」を付けるかどうかですね。

Reply:
ありがとうございます。熟語でしたか。勉強になりました。
せっかく変更になっていますので、「万が一」を加えていただけると良いかな、と思いました。

Reply:
そうですね、そのようにします。